urcu is needed for cds_list_for_each_entry_rcu (rcu_dereference_sym symbol)
--- lttng-ust-2.13.1/src/lib/lttng-ust-ctl/Makefile.am.orig	2021-12-09 20:45:58.000000000 +0100
+++ lttng-ust-2.13.1/src/lib/lttng-ust-ctl/Makefile.am	2022-02-05 13:08:04.746973171 +0100
@@ -16,4 +16,5 @@ liblttng_ust_ctl_la_LIBADD = \
 	$(top_builddir)/src/common/libcounter-clients.la \
 	$(top_builddir)/src/common/libustcomm.la \
 	$(top_builddir)/src/common/libcommon.la \
+	$(URCU_LIBS) \
 	$(DL_LIBS)
